Source/Csla.AspNetCore/Blazor/State/StateController.cs

```diff
@@ -65,25 +76,44 @@ public virtual byte[] Get()
     [HttpPut]
     public virtual void Put(byte[] updatedSessionData)
     {
-      var formatter = new MobileFormatter(ApplicationContext);
+      var formatter = Csla.Serialization.SerializationFormatterFactory.GetFormatter(ApplicationContext);
       var buffer = new MemoryStream(updatedSessionData)
       {
         Position = 0
       };
       var session = (Session)formatter.Deserialize(buffer);
-      session.IsCheckedOut = false;
       _sessionManager.UpdateSession(session);
     }
+  }
 
+  /// <summary>
+  /// Message type for communication between StateController
+  /// and the Blazor wasm client.
+  /// </summary>
+  public class StateResult
```

Comment:
Shouldn't this type be in it's own file?

Reply:
Yes